oidc-api/src/main/java/uk/gov/di/authentication/oidc/lambda/LogoutHandler.java

```diff
@@ -198,6 +207,7 @@ public APIGatewayProxyResponseEvent logoutRequestHandler(APIGatewayProxyRequestE
                     Optional.of(clientID));
         }
 
+        var finalAuditUser = auditUser;
```

Comment:
Probably don't need this?

Reply:
<img width="484" alt="image" src="https://github.com/govuk-one-login/authentication-api/assets/137415037/45aec94d-fef9-4374-b4ae-e06c75ee70b9">
I added it in as IntelliJ was getting upset about it not being final or effectively final